packages/app/src/cli/services/dev/processes/draftable-extension.ts

```diff
@@ -38,8 +44,37 @@ export const pushUpdatesForDraftableExtensions: DevProcessFunction<DraftableExte
     }),
   )
 
+  const appConfigExtensions = extensions.filter((extension) => extension.isConfigExtension)
+  const normalExtensions = extensions.filter((extension) => !extension.isConfigExtension)
+
+  if (appConfigExtensions.length > 0) {
+    await Promise.all(
+      createAppConfigExtensionPathsMap(remoteExtensions, appConfigExtensions)
+        .map(async ([appConfigPath, appConfigExtension]) => {
+          if (!appConfigExtension) {
+            return
+          }
+
+          // Create a single watcher for each app config file and update all related extensions when the config updates
+          return [
```

Comment:
wait, all config extensions read from the app.toml right? shouldn't we watch just once that file? doesn't make sense to add multiple watchers to the same file. Or am I missing anything?